python/scope.py

```diff
@@ -1048,10 +1048,12 @@ async def eval_command(self, tree):
 
                 scope.assign_to_pattern(pattern, i, certain=True)
                 exit, value = await scope.eval_command(code)
-                if exit == True: # exit can be a string
-                    return True, value
                 if exit == "continue":
                     continue
+                if exit:
+                    if value == None:
+                        return False, None
```

Comment:
Hmm, wouldn't a better solution be to do what you did with continue and have it return `"break"`?

Reply:
Added